packages/frontend/editor-ui/src/components/ResourceMapper/MappingFields.vue

```diff
@@ -449,7 +449,7 @@ defineExpose({
 	padding: 0 0 0 var(--spacing-s);
```

Comment:
Looks really nice. Here's a nit pick: If it gives the desired result, we can keep `width: 100%` and adjust the horizontal padding here to keep contents inside:

```suggestion
	padding: 0 var(--spacing-s);
```

Reply:
Hmm that doesn't quite make it line up:

<img width="238" alt="image" src="https://github.com/user-attachments/assets/095a52f8-51a8-46d0-83d6-042669c42746" />
(With the width changed back)